Scripts/run-oclint.sh

```diff
@@ -9,51 +9,53 @@ xcodebuild_log_path=${temp_dir}/xcodebuild.log
 
 hash oclint &> /dev/null
 if [ $? -eq 1 ]; then
-    echo >&2 "oclint not found, analyzing stopped"
+    echo >&2 "[OCLint] oclint not found, analyzing stopped"
     exit 1
 fi
 
 oclint --version
 
-echo "[*] cleaning up generated files"
+echo "[OCLint] cleaning up generated files"
 [[ -f $compile_commands_path ]] && rm ${compile_commands_path}
 [[ -f $xcodebuild_log_path ]] && rm ${xcodebuild_log_path}
 
-echo "[*] starting xcodebuild to build the project.."
+echo "[OCLint] starting xcodebuild to build the project.."
 if [ -d WordPress.xcworkspace ]; then
-    echo "[*] we're running the script from the CLI"
+    echo "[OCLint] we're running the script from the CLI"
     xcode_workspace="WordPress.xcworkspace"
+    oclint_args+=" -report-type=html -o=oclint_result.html"
     pipe_command=""
 elif [ -d ../WordPress.xcworkspace ]; then
-    echo "[*] we're running the script from Xcode"
+    echo "[OCLint] we're running the script from Xcode"
     xcode_workspace="../WordPress.xcworkspace"
     pipe_command="| sed 's/\\(.*\\.\\m\\{1,2\\}:[0-9]*:[0-9]*:\\)/\\1 warning:/'"
 else
     # error!
-    echo >&2 "workspace not found, analyzing stopped"
+    echo >&2 "[OCLint] workspace not found, analyzing stopped"
     exit 1
 fi
 
-echo "[*] cleaning project"
+echo "[OCLint] cleaning project"
 xctool clean \
            -sdk "iphonesimulator8.1" \
            -workspace $xcode_workspace -configuration Debug -scheme WordPress \
            CONFIGURATION_BUILD_DIR=$build_dir \
            DSTROOT=$build_dir OBJROOT=$build_dir SYMROOT=$build_dir \
+           reporter pretty \
            > ${temp_dir}/clean.log
 
-echo "[*] building project"
+echo "[OCLint] building project"
 xctool build \
            -sdk "iphonesimulator8.1" \
            CONFIGURATION_BUILD_DIR=$build_dir \
            -workspace $xcode_workspace -configuration Debug -scheme WordPress \
            DSTROOT=$build_dir OBJROOT=$build_dir SYMROOT=$build_dir \
            -reporter json-compilation-database:$compile_commands_path
-           #| tee $xcodebuild_log_path
+           
 
 if [ $TRAVIS ]; then
-    echo "[*] only files changed on push";    
-    include_files=`git diff $TRAVIS_COMMIT_RANGE --name-only | grep '\.m' | tr '\n' ' -i '`
+    echo "[OCLint] only files changed on push";    
+    include_files=`git diff $TRAVIS_COMMIT_RANGE --name-only | grep '\.m' | tr '\n' '|' | sed 's/|*$/"/g'`
```

Comment:
Maybe I'm reading this wrong, but won't this only have an end `"` and not a starting one? e.g. I did `git diff ee020a2 --name-only  | tr '\n'g'|' |  sed 's/|*$/"/g` locally and got `Podfile|Podfile.lock"`